ansible/roles/mysql/tasks/main.yml

```diff
@@ -22,7 +22,7 @@
 
 - name: mysql | Create db_dumps directory
   sudo: yes
-  file: path={{ rox.root_folder }}/db_dumps state=directory group="www-data" mode="g+rw"
+  file: path={{ rox.root_folder }}/db_dumps state=directory group="www-data" mode="664"
```

Comment:
ansible doesn't support, at least in the apt-get version available, the "g+rw" way of defining permissions. Changing to octal prevents this error:

<img width="1267" alt="screenshot 2016-01-28 22 49 49" src="https://cloud.githubusercontent.com/assets/1557348/12697756/f193f618-c758-11e5-8bac-c4657f6b0190.png">